OMCompiler/Compiler/runtime/Makefile.omdev.mingw

```diff
@@ -1,26 +1,16 @@
-# Adrian Pop, adrpo@ida.liu.se, 2006-02-01
-# Makefile for Compiler/runtime compilation of OMC using OMDev-mingw
-# OMDev-mingw: http://www.ida.liu.se/~adrpo/omc/omdev/
-#
-# $Id: Makefile.omdev.mingw.in 1817 2006-02-01 12:21:26Z adrpo $
-#
+# Makefile for Compiler/runtime compilation of OMC using MSYS UCRT
 
-# The path to the OMDev-mingw package MUST BE SET!
-#OMDEV=$OMDEV
 # Test if the needed variables are there...
 .testvariables:
 ifndef OMDEV
 	@echo You have to set the OMDEV variabile pointing to your OMDev package root! Exiting....
-	@echo Take the OMDev package from: http://www.ida.liu.se/~adrpo/omc/omdev/
-	@echo For questions or problems email Adrian Pop, adrpo@ida.liu.se
+	@echo See OpenModelica/OMCompiler/README.Windows.md
 	ABORT
 endif
 
-OMDEVMSYS=$(shell cygpath $$OMDEV)
-
 TOP_DIR=../../
 USE_METIS = -DUSE_METIS
-CORBAHOME = $(OMDEVMSYS)
+CORBAHOME = $(OMDEV)
```

Comment:
In the main Makefile.omdev.mingw I added

```Makefile
OMDEV = $(shell cygpath $$OMDEV)
```

So with a bit of luck this should already be correct. Otherwise we can add this here as well.

Reply:
The problem is that you need both OMDev in the original form (to use with MSVC) and OMDEVMSYS in the msys2 form to use in the makefiles.